composer.json

```diff
@@ -23,6 +23,9 @@
     "require": {
         "php": ">=5.2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "4.*"
```

Comment:
Maybe consider `4.*|5.*` because, officially, phpunit 4 is marked as not compatible with php 7, only phpunit 5 is.